pkg/api/vlabs/const.go

```diff
@@ -107,3 +107,36 @@ const (
 	// AgentPoolProfileRoleInfra is the infra role
 	AgentPoolProfileRoleInfra AgentPoolProfileRole = "infra"
 )
+
+const (
+	//AzureEdgeDCOSBootstrapDownloadURL is the azure edge CDN download url
+	AzureEdgeDCOSBootstrapDownloadURL = "https://dcosio.azureedge.net/dcos/%s/bootstrap/%s.bootstrap.tar.xz"
+	//AzureChinaCloudDCOSBootstrapDownloadURL is the China specific DCOS package download url.
+	AzureChinaCloudDCOSBootstrapDownloadURL = "https://acsengine.blob.core.chinacloudapi.cn/dcos/%s.bootstrap.tar.xz"
+	//AzureEdgeDCOSWindowsBootstrapDownloadURL
+)
+
+const (
+	// AzureCniPluginVerLinux specifies version of Azure CNI plugin, which has been mirrored from
+	// https://github.com/Azure/azure-container-networking/releases/download/${AZURE_PLUGIN_VER}/azure-vnet-cni-linux-amd64-${AZURE_PLUGIN_VER}.tgz
+	// to https://acs-mirror.azureedge.net/cni
+	AzureCniPluginVerLinux = "v1.0.15"
```

Comment:
This should be v1.0.16

Reply:
in the setCustomCloudProfileDefaults() of src/github.com/Azure/aks-engine/pkg/api/defaults.go

if CustomCloudProfile is in config, but AzureEnvironmentSpecConfig is empty, 

The default AzureStackCloudSpec will be used, it will persist into the apimobel.json 

		} else {
			customCloudDefault := AzureCloudSpecEnvMap[AzureStackCloud]
			p.CustomCloudProfile.AzureEnvironmentSpecConfig = &customCloudDefault
		}

Reply:
On the second look, the default value of AzureEnvironmentSpecConfig under vlabs are not used. So I removed them.